jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/CookieParser.java

```diff
@@ -52,6 +52,8 @@ default void parseFields(List<String> rawFields) throws InvalidCookieException
             parseField(field);
     }
 
+    CookieCompliance getCookieCompliance();
```

Comment:
javadoc

Reply:
Easier to just get rid of this method, it's no longer being used anyway.